src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -80,7 +83,17 @@ public class RepositoryCloneViewModel : BaseViewModel, IRepositoryCloneViewModel
                 signalReset: filterResetSignal
             );
 
-            BaseRepositoryPathValidator = this.CreateBaseRepositoryPathValidator();
+            var baseRepositoryPath = this.WhenAny(
+                x => x.BaseRepositoryPath,
+                x => x.SelectedRepository,
+                (x, y) => x.Value);
+
+            BaseRepositoryPathValidator = ReactivePropertyValidator.ForObservable(baseRepositoryPath)
+                .IfNullOrEmpty("Please enter a repository path")
+                .IfTrue(x => x.Length > 200, "Path too long")
+                .IfContainsInvalidPathChars("Path contains invalid characters")
+                .IfPathNotRooted("Please enter a valid path")
+                .IfTrue(IsAlreadyRepoAtPath, Resources.RepositoryNameValidatorAlreadyExists);
```

Comment:
The only two differences are that this one takes an observable and the other one makes its own observable, so we can just pass in one instead of letting it do that; and that there's an extra `IsTrue` directive on this one, which can be added to the extension method since both create and clone should be doing the same validations. Everything else is the same, am I missing something?